### ISSUE TITLE
command 'language-julia.executeJuliaCodeInREPL' not found error

### ISSUE THREAD

**Issue by optixlab** [julia-vscode/julia-vscode#412]
Newly installed VS Code + Julia extension. The first few times worked like a charm with user settings indicating the path to Julia.exe. The usual behavior was to open the folder in VS Code and then open the .jl file. <CTRL>-<Enter> would start the REPL and execute. 

Issue: Keybinding does not work. 

> **Warn: command 'language-julia.executeJuliaCodeInREPL' not found**

I tried other key-bindings, such as <ALT>-<Enter>, with similar result:

> **Warn: command 'language-julia.executeJuliaBlockInREPL' not found**

**Comment by davidanthoff** [comment]
What version of the extension are you running?

**Comment by davidanthoff** [comment]
@optixlab is that still a problem?

**Comment by ChristianKurz** [comment]
I got this problem just now. I don't recall changing anything, worked like a charm yesterday.

**Comment by davidanthoff** [comment]
Hm, that is very strange... It would be great if we could somehow reproduce this...

**Comment by ChristianKurz** [comment]
I actually only have the problem using `version 0.10.2`.

Error msg is `command 'language-julia.startREPL' not found`
in addition to those mentioned above.

While code highlighting of julia files works, linting doesn't seem to.

I already reinstalled VSCode and the julia extension, but to no avail.

**Comment by davidanthoff** [comment]
That is really, really strange. The only difference between 0.10.1 and 0.10.2 is a super minor change to how we auto-detect julia on Macs...

**Comment by dingbo8128** [comment]
I also get this problem using the latest dev version 0.11.0-dev on windows 10.

**Comment by arnaudsj** [comment]
I am getting the same issue on Debian buster on a Chromebook. Julia 1.0.3 installed via apt. Any way to see the logs to see what triggers it? I am using the master branch in Github as of today (commit #73489c2a7a31fc04a9fb8ea6874eb9d4516a62fd)

**Comment by davidanthoff** [comment]
I believe this should have been fixed a while ago, closing.

**Comment by Nosferican** [comment]
Getting the same issue with v0.16.11.

**Comment by jdkealy** [comment]
I just had this same issue. It worked great. Restarted my computer and it doesn't work.

**Comment by tbenst** [comment]
Same issue for me on Julia-vscode 1.2.8 and vscode 1.58.2. I'm using `remote-ssh` over a very slow internet connection (500ms ping; 0.05Mbps while traveling. I had to increase timeout for vscode remote-ssh in order to connect. Perhaps there is an underlying timeout condition that is causing these symptoms..? I did not notice this problem until switching to slow internet while traveling. Alternatively, I did recently update so perhaps slow internet is a red herring, but mentioning in case useful
